### PR TITLE
[docs] add icons to admonition titles

### PR DESCRIPTION
**Pull request by chrisjsewell**
and also replace ad-hoc "more" icon in quickstart, with using `seealso` admonition

see e.g.: https://sphinx--12486.org.readthedocs.build/en/12486/usage/installation.html

---

before:

image

after:

image

### CODE REVIEW

File: doc/usage/quickstart.rst [hunk 1]
```diff
@@ -308,7 +324,3 @@ More topics to be covered
 .. [#] This is the usual layout.  However, :file:`conf.py` can also live in
        another directory, the :term:`configuration directory`.  Refer to the
        :doc:`sphinx-build man page </man/sphinx-build>` for more information.
-
-.. |more| image:: /_static/more.png
-          :align: middle
-          :alt: more info
```
Comment (jayaddison) [inline doc/usage/quickstart.rst:314]: This code removal puzzled me for a few moments; now I understand it's removal of a substitution definition that is no-longer-used on the page (a bit like an unused import).

**Review by jayaddison — APPROVED** [review]
Looks good to me :+1:

**Comment by jfbu** [comment]
+1 I will copy these settings (which adds to #12439) and make them default for admonitions in PDF output (perhaps not yet for the icons, which however is possibly via `fontawesome5` LaTeX package I think).  Although the possibility exists since 5.1.0 and 6.2.0 for user to customize rendering of admonitions, the defaults for PDF have remained the rather boring legacy old ones so far.  A LaTeX interface has to be added for the coloured title.  As I said icons are possible too, but add a font dependency.  As you use HSB this will gave us an excuse to make Sphinx require `xcolor` LaTeX package which helps simplify a bit some internal code which tested whether `xcolor` was available or not.

**Comment by jfbu** [comment]
@chrisjsewell I have tried to transfer this to PDF at #12508.

I tried to use exact same choices for the colours of title rows.

For the icons also I followed choices here except I was not able to find the fontawesome5 name for "spark" (used by caution and danger, so I used a "radiation" icon) and besides I used a "skull-crossbones" for error admonition in place of a "times-circle".

I am making this comment here to ask what you would think of using shapes such as [lightbulb](https://fontawesome.com/icons/lightbulb?f=classic&s=solid) rather than question mark for hint and tip, and of using [share](https://fontawesome.com/icons/share?f=classic&s=solid) for seealso ? I was told the [pydata theme](https://pydata-sphinx-theme.readthedocs.io/en/stable/) does something like that or similar (and actually their [RTD home](https://pydata-sphinx-theme.readthedocs.io/en/stable/) has a Seealso with the "share" icon I was hinting at).

**Comment by chrisjsewell** [comment]
Yeh cheers, I'm certainly open to suggestions 👍

**Comment by chrisjsewell** [comment]
Thebiconsni used were simply copy/pasted from furo, if I recall correctly

**Comment by jfbu** [comment]
ok, things for me are simple for PDF output I only need to use the name of an icon as provided by LaTeX fontawesome5 package, I am not using svg images.

**Pull request merged**